docs/docs/04-standard-library/compatibility/compatibility.json

```diff
@@ -53,71 +53,71 @@
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false, "issue": 1282 },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": true }
     },
     "putJson": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false, "issue": 1282 },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": true }
     },
     "get": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false, "issue": 1282 },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": true }
     },
     "tryGet": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": false }
     },
     "getJson": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": false }
     },
     "tryGetJson": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": false }
     },
     "delete": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false, "issue": 1282 },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": true }
     },
     "tryDelete": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": false }
     },
     "exists": {
       "sim": { "implemented": true },
       "tf-aws": { "implemented": true },
       "tf-azure": { "implemented": true },
-      "tf-gcp": { "implemented": false },
+      "tf-gcp": { "implemented": true },
       "awscdk": { "implemented": false }
     },
     "metadata": {
-      "sim": { "implemented": false },
-      "tf-aws": { "implemented": false },
-      "tf-azure": { "implemented": false },
-      "tf-gcp": { "implemented": false },
-      "awscdk": { "implemented": false }
+      "sim": { "implemented": true },
+      "tf-aws": { "implemented": true },
+      "tf-azure": { "implemented": true },
+      "tf-gcp": { "implemented": true },
+      "awscdk": { "implemented": true }
```

Comment:
I don't think this is true...

Reply:
Fixed that adding the issue numbers as well.